packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -165,7 +165,7 @@ function FontLibraryProvider( { children } ) {
 	// Theme data
 	const { site, currentTheme } = useSelect( ( select ) => {
 		return {
-			site: select( coreStore ).getSite(),
```

Comment:
I need to create an alias for `getSite` and friends for backward compatibility.

Reply:
I came back to a similar idea. Might need to merge #59881 first thought :)

Reply:
Resolved in [ae8e941](https://github.com/WordPress/gutenberg/pull/59243/commits/ae8e941ecf88a9a7b60a7b7b635d40fb03e551c6).